netket/vqs/mc/kernels.py

```diff
@@ -68,7 +69,7 @@ def local_value_kernel_jax(
     """
     σp, mel = O.get_conn_padded(σ)
     logpsi_σ = logpsi(pars, σ)
-    logpsi_σp = logpsi(pars, σp)
+    logpsi_σp = wrap_to_support_multiple_batch_axes(logpsi)(pars, σp)
```

Comment:
Now that I think about it. I am not in favour of this change here. 
The `apply_fun` should be wrapped, in case, inside of the variational state.

Otherwise we end up with a complex patchwork of what supports and what does not support multiple axes. 

My idea is that those operators & co should have the simplest as possible code, and then we can put a lot of those smarts in the constructors of MCState or similar.
If a user develops/uses his own variational state, he can wrap the callable he uses as apply fun by himself. 

Again, the depths of netket's operators should stay as simple as possible..